server/plugin/virtual_agent.go

```diff
@@ -329,13 +328,18 @@ func (p *Plugin) ProcessResponse(data []byte) error {
 				}
 			}
 		case *OutputImage:
-			var post *model.Post
-			post, err = p.CreateOutputImagePost(res, userID)
-			if err != nil {
-				return err
+			linkContents := strings.Split(res.Value, "/")
+			if len(linkContents) < 1 {
+				if _, err = p.DM(userID, res.AltText); err != nil {
```

Comment:
```suggestion
				if _, err = p.DM(userID, fmt.Sprintf("Image: %s", res.AltText)); err != nil {
```

in case we are just showing the alt attribute text of the image then I think we should also mention that it is for an image